src/validation.cpp

```diff
@@ -837,6 +837,12 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
         return state.Invalid(TxValidationResult::TX_INPUTS_NOT_STANDARD, "bad-txns-nonstandard-inputs");
     }
 
+    if (tx.GetTotalSize() > m_pool.m_max_dummy_script_bytes) {
```

Comment:
`m_max_dummy_script_bytes` is still defined as `optional<unsigned>` ; should presumably just be `unsigned`, but if not, shouldn't be compared to things without first checking it's not `nullopt`

Reply:
An oversight on my part thank you for reporting it.